Makefile

```diff
@@ -20,6 +20,8 @@ ifeq ($(INTERACTIVE), 1)
 	DOCKER_FLAGS += -t
 endif
 DOCKER_RUN_DOCKER := $(DOCKER_FLAGS) "$(DOCKER_IMAGE)"
+# Use =, not :=, so that this is not expanded when we are not running unit tests.
+UNIT_TEST_PACKAGES = $(shell find -name '*_test.go' | sed 's,/[^/]*$$,,' | sort -u | grep -v '^\./\(integration\|vendor\)$$')
```

Comment:
Getting `go list` to work was surprisingly non-trivial (e.g. it doesn't work within `sudo` when `GOPATH` is not set), but it allowed to drop the "empty package list" case and generally simplified the `Makefile`.

Thanks!